### PR TITLE
Support for different configuration profiles for command generation

### PR DESCRIPTION
**Pull request by vidyas00**
Adding support to generate [ntttcp] commands for multiple configurations - ex: generating commands for sampling (default)) or for testing

@omarcardona @cmaloney-msft @tmolenh

### CODE REVIEW

**Comment by vidyas00** [comment]
Tested on Azure VMs with Default and Custom Params

**Comment by vidyas00** [comment]
ping

**Comment by vidyas00** [comment]
Two changes:
1) Adding support to generate [ntttcp] commands for multiple configurations - ex: generating commands for sampling (default)) or for testing
2) Fixing bug with latte command generation (where -dump cannot be specified for time based tests)

@omarcardona @cmaloney-msft @tmolenh

File: PERFTEST.PS1 [hunk 1]
```diff
@@ -16,11 +17,12 @@ function input_display {
...
     Write-Host "$g_path\$scriptName"
     Write-Host " Inputs:"
-    Write-Host "  -Detail = $Detail"
-    Write-Host "  -Diag   = $Diag"
-    Write-Host "  -DestIp = $DestIp"
-    Write-Host "  -SrcIp  = $SrcIp"
-    Write-Host "  -OutDir = $OutDir"
+    Write-Host "  -Detail        = $Detail"
+    Write-Host "  -Diag          = $Diag"
+    Write-Host "  -Configuration = $Configuration"
```
Comment (omarcardona) [inline PERFTEST.PS1:22]: Please rename to config throughout.  To seem the variable sizes withing a reasonable range.

File: ntttcp/ntttcp.TESTGEN.ps1 [hunk 2]
```diff
@@ -109,12 +111,13 @@ function banner {
...
-        [parameter(Mandatory=$true)] [String] $OutDir
+        [parameter(Mandatory=$true)] [String] $OutDir,
+        [parameter(Mandatory=$true)] [ValidateScript({Test-Path $_})] [String] $ConfigFile
     )
 
-    # execution time in seconds
-    [int] $g_runtime = 10
-    [int] $g_ptime   = 2
+    #Load the variables needed to generate the commands
+    . .\$ConfigFile
```
Comment (omarcardona) [inline ntttcp/ntttcp.TESTGEN.ps1:119]: What is the config file is not available/missing? 
Should there be safe defaults?  
Should we generate an error?  
Does the documentation detail that a config file and associated format is required?

**Pull request closed without merging**